docs/proposals/proposal-001-trigger-and-deploy.md

```diff
@@ -57,7 +57,7 @@ multiple configurations of Falco and more CNCF projects as they are onboarded.
 
 ### Goals
 
-- Trigger the pipeline when a new release of a project happens
+- Trigger the pipeline once a day for the latest version of the project
```

Comment:
@rossf7 what about we trigger the workflow for the benchmarking in a specific time period where it is cheap and lower load on grid like at night may be where the k8s cluster is located
WDYT?

Reply:
@dipankardas011 Thanks good to consider this. I updated the spec to say 08:00 UTC.

This is because for renewable energy I think the best time to run it is in daylight hours when demand is low. As there should be solar energy available then. 

Although the cluster is in France so the average carbon intensity is low because the grid has a lot of nuclear.